auctions/english-auction/src/main.sw

```diff
@@ -0,0 +1,253 @@
+contract;
+
+dep data_structures/state;
+dep errors;
+dep events;
+dep interface;
+dep utils;
+
+// TODO: Move these into alphabetical order once https://github.com/FuelLabs/sway/issues/409 is resolved
+dep data_structures/auction_asset;
+dep data_structures/auction;
+
+use auction_asset::AuctionAsset;
+use auction::Auction;
+use errors::{AccessError, InitError, InputError, UserError};
+use events::{BidEvent, CancelAuctionEvent, CreateAuctionEvent, WithdrawEvent};
+use interface::{EnglishAuction, NFT};
+use state::State;
+use std::{
+    block::height,
+    chain::auth::{
+        AuthError,
```

Comment:
Doesn't seem to be used, is this needed? I may have asked this already - or in another PR.